### PR TITLE
Build manifest for net.sourceforge.btanks

### PR DESCRIPTION
**Pull request by cosimoc**
Based on the Freedesktop 1.6 SDK.

### CODE REVIEW

File: net.sourceforge.btanks.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    "rename-icon": "btanks",
+    "modules": [
+        "shared-modules/glu/glu-9.0.0.json",
+        "shared-modules/SDL/SDL-1.2.15.json",
+        "shared-modules/smpeg/smpeg-0.4.5.json",
+        "shared-modules/SDL/SDL_image-1.2.12.json",
+        "shared-modules/lua5.3/lua-5.3.2.json",
+        {
+            "name": "scons",
+            "no-autogen": true,
```
Comment (Barthalion) [inline net.sourceforge.btanks.json]: Wouldn't `"buildsystem": "simple"` be better here?

**Comment by Barthalion** [comment]
Test build: https://flathub.org/builds/#/builders/5/builds/241

File: net.sourceforge.btanks.json (hunk 1 above)
Comment (cosimoc) [inline net.sourceforge.btanks.json]: Yeah, that could work too, but that feature did not exist when I wrote this manifest :-)
Let's see if the rest of the build completes -- if it does perhaps we can just fix it later once we have a repo.

**Comment by cosimoc** [comment]
Build was successful!

**Comment by Barthalion** [comment]
[Merged](https://github.com/flathub/net.sourceforge.btanks), thanks!

**Pull request closed without merging**